cli/stork-publisher-agent.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	storkpublisheragent "github.com/Stork-Oracle/stork_external/stork-publisher-agent"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/pkgerrors"
+	"github.com/spf13/cobra"
+	"net/http"
+	"regexp"
+	"time"
+)
+
+var Hex32Regex = regexp.MustCompile(`^0x[0-9a-fA-F]+$`)
+
+var publisherAgentCmd = &cobra.Command{
+	Use:   "publisher-agent",
+	Short: "Start a process to sign price updates and make them available to Stork",
+	RunE:  runPublisherAgent,
+}
+
+// required
+const SignatureTypeFlag = "signature-type"
+const OracleIdFlag = "oracle-id"
+const PrivateKeyFlag = "private-key"
```

Comment:
Stark and Evm keys are differently structured pretty sure. Flagging that this is an issue, lets discuss offline.